clusterloader2/testing/load/config.yaml

```diff
@@ -0,0 +1,155 @@
+automanagedNamespaces: 3
+tuningSets:
+- name: Uniform100qps
+  qpsLoad:
+    qps: 100
+- name: Uniform5qps
+  qpsLoad:
+    qps: 5
+steps:
+- measurements:
+  - Identifier: GlobalTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: CreatePodsTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+# Create RCs
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: Uniform100qps
+    objectBundle:
+    - basename: service
+      objectTemplatePath: service.yaml
+      templateFillMap:
+        RCBasename: rc
+    - basename: rc
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        ReplicasMin: 30
+        ReplicasMax: 30
+- measurements:
+  - Identifier: WaitForRunningRCs
+    Method: WaitForRunningRCs
+    Params:
+      labelSelector: kind = loadRC
+      timeout: 2m
+- measurements:
+  - Identifier: CreatePodsTimer
+    Method: Timer
+    Params:
+      action: stop
+      label: Pods creation
+  - Identifier: ScaleRCsTimer1
+    Method: Timer
+    Params:
+      action: start
+# Scale RCs first time
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: Uniform5qps
+    objectBundle:
+    - basename: service
+      objectTemplatePath: service.yaml
+      templateFillMap:
+        RCBasename: rc
+    - basename: rc
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        ReplicasMin: 15
+        ReplicasMax: 45
+- measurements:
+  - Identifier: WaitForScaledRCsRunning1
+    Method: WaitForRunningRCs
+    Params:
+      labelSelector: kind = loadRC
+      timeout: 2m
+- measurements:
+  - Identifier: ScaleRCsTimer1
+    Method: Timer
+    Params:
+      action: stop
+      label: Scaling RCs first time
+  - Identifier: ScaleRCsTimer2
+    Method: Timer
+    Params:
+      action: start
+# Scale RCs second time
```

Comment:
You can get rid of the second round of scaling which we just changed - https://github.com/kubernetes/kubernetes/pull/67769

Reply:
Done.